frontend/packages/ceph-storage-plugin/src/components/modals/add-capacity-modal/add-capacity-modal.tsx

```diff
@@ -94,23 +94,26 @@ export const AddCapacityModal = withHandlePromise((props: AddCapacityModalProps)
           <div className="ceph-add-capacity_sc-dropdown">
             <OCSStorageClassDropdown onChange={setStorageClass} defaultClass={storageClass} />
           </div>
-          <label className="control-label" htmlFor="requestSize">
-            Raw Capacity
-            <FieldLevelHelp>{labelTooltip}</FieldLevelHelp>
-          </label>
-          <div className="ceph-add-capacity__form">
-            <input
-              className={classNames('pf-c-form-control', 'ceph-add-capacity__input')}
-              type="number"
-              name="requestSize"
-              value={osdSizeWithoutUnit}
-              required
-              disabled
-            />
-            <div className="ceph-add-capacity__input--info-text">
-              x 3 replicas = <strong>{provisionedCapacity} TiB</strong>
-            </div>
-          </div>
+          {osdSizeWithoutUnit && (
+            <>
+              <label className="control-label" htmlFor="requestSize">
+                Raw Capacity
+                <FieldLevelHelp>{labelTooltip}</FieldLevelHelp>
+              </label>
+              <div className="ceph-add-capacity__form">
+                <input
```

Comment:
Can we add a data-test-id to this input?

Reply:
Its getting added in my PR. So, if we want, we can skip it here.

Reply:
https://github.com/openshift/console/pull/5016/files#diff-ecff7031320d48d9e801ba99a2fb7188R123

Reply:
👍